config_defaults_inc.php

```diff
@@ -976,6 +976,18 @@
 	$g_show_avatar_threshold = DEVELOPER;
 
 	/**
+	 * To host your own avatar site change this setting
+	 * @global string $g_show_avatar_url
+	 */
+	$g_show_avatar_url = 'http://www.gravatar.com';
```

Comment:
I would rather if we have one config option called $g_show_avatar_url.  We default it in config_defaults_inc.php based on ssl vs. not.  Then everywhere in the code we just get the value of the config and use it.  If an instance wants to override that, they will just have one option to override and they will have to set it to a value that matches they https vs. http scheme.  The documentation of the config option should clarify that https avatar site is needed if MantisBT itself is being accessed via https, otherwise, there will be warnings of unsecure content.